pkg/kepctl/kepctl.go

```diff
@@ -181,3 +263,73 @@ func (c *Client) writeKEP(kep *keps.Proposal, opts CommonArgs) error {
 	fmt.Fprintf(c.Out, "writing KEP to %s\n", newPath)
 	return ioutil.WriteFile(newPath, b, os.ModePerm)
 }
+
+type PrintConfig interface {
+	Title() string
+	Format() string
+	Value(*keps.Proposal) string
+}
+
+type printConfig struct {
+	title     string
+	format    string
+	valueFunc func(*keps.Proposal) string
+}
+
+func (p *printConfig) Title() string  { return p.title }
+func (p *printConfig) Format() string { return p.format }
+func (p *printConfig) Value(k *keps.Proposal) string {
+	return p.valueFunc(k)
+}
+
+var dfltConfig = map[string]printConfig{
+	"Authors":     {"Authors", "%-30s", func(k *keps.Proposal) string { return strings.Join(k.Authors, ", ") }},
+	"LastUpdated": {"Updated", "%-10s", func(k *keps.Proposal) string { return k.LastUpdated }},
+	"SIG": {"SIG", "%-12s", func(k *keps.Proposal) string {
+		if strings.HasPrefix(k.OwningSIG, "sig-") {
+			return k.OwningSIG[4:]
+		} else {
+			return k.OwningSIG
+		}
+	}},
+	"Stage":  {"Stage", "%-6s", func(k *keps.Proposal) string { return k.Stage }},
+	"Status": {"Status", "%-16s", func(k *keps.Proposal) string { return k.Status }},
+	"Title":  {"Title", "%-30s", func(k *keps.Proposal) string { return k.Title }},
+}
+
+func DefaultPrintConfigs(names ...string) []PrintConfig {
+	var configs []PrintConfig
+	for _, n := range names {
+		// copy to allow it to be tweaked by the caller
+		c := dfltConfig[n]
+		configs = append(configs, &c)
+	}
+	return configs
+}
+
+func (c *Client) PrintTable(configs []PrintConfig, proposals []*keps.Proposal) {
```

Comment:
Let's opt to use a library here: https://github.com/olekukonko/tablewriter
There's some example usage in k/release: https://cs.k8s.io/?q=github.com%2Folekukonko%2Ftablewriter&i=nope&files=&repos=kubernetes/release